internal/cmd/run/cmd.go

```diff
@@ -182,3 +220,24 @@ func (r *run) run(cmd *cobra.Command) {
 		os.Exit(1)
 	}
 }
+
+// exitIfUnsupported prints an error containing unsupported field names and exits
+// if any of those fields are not their default values.
+func exitIfUnsupported(options manager.Options) {
+	var keys []string
+	// The below options are webhook-specific, which is not supported by ansible.
```

Comment:
@joelanford do you mean that we create a separate config rather than using controller-runtime's [manager.Config](https://github.com/kubernetes-sigs/controller-runtime/blob/v0.10.0/pkg/manager/manager.go#L99)?

Reply:
I think its fine to keep using `manager.Options`, but I got the impression that this check was necessary because the `manager.Options` could be somehow loaded or configured by something provided by the user to contain these values.

Since we control the `run` subcommand, I was thinking that we can make it impossible for the `manager.Options` struct to have these values in the first place by ensuring there are no flags or config files that would permit those values being set in the first place.

Reply:
we allow users to provide a controller-runtime [config](https://github.com/operator-framework/helm-operator-plugins/pull/101/files#diff-c572e55a50737371783ccb39ea800029a4a3343ba47f46f60da0037dcb68e25cR87) file’s path through a flag with which users could set the webhook port, host, certDir etc. We do allow users to set a few other values like MetricsBindAddress through the c-r config [file](https://github.com/kubernetes-sigs/controller-runtime/blob/f20692b7ce2a9dda7eb298504a9c47aaf5a96544/pkg/manager/manager.go#L397). Do you think we should remove that flag entirely and not let users to pass the config file ? That would be a breaking change I suppose.